doc/release_notes/release_notes_3.1.rst

```diff
@@ -92,29 +92,16 @@ Improved TSC frequency reporting
 Upgrading to v3.1 from Previous Releases
 ****************************************
 
-As with the v3.0 release With the introduction of the Configurator UI tool, the need for manually editing
-XML files is gone.  While working on this improved Configurator, we've also made
-many adjustments to available options in the underlying XML files, including
-merging the previous scenario and launch XML files into a combined scenario XML
-file.  The board XML file generated by the v3.1 Board Inspector tool includes
-more information about the target system that is needed by the v3.1
-Configurator.
-
-We recommend you generate a new board XML for your target system with the v3.1
-Board Inspector and use the v3.1 Configurator to generate a new
-scenario XML file and launch scripts. Board XML and Scenario XML files
-created by previous ACRN versions will not work with the v3.1 ACRN hypervisor
-build process and could produce unexpected errors during the build.
-
-Given the scope of changes for the v3.1 release, we have recommendations for how
-to upgrade from prior ACRN versions:
+We recommend you generate a new board XML for your target system with the v3.1 Board Inspector. You should also use the v3.1 Configurator to generate a new scenario XML file and launch scripts. Scenario XML files and launch scripts created by previous ACRN versions will not work with the v3.1 ACRN hypervisor build process and could produce unexpected errors during the build.
```

Comment:
Can you keep the line length at 80 characters max (as per https://projectacrn.github.io/latest/developer-guides/doc_guidelines.html#tabs-spaces-and-indenting)

Reply:
I will fix it soon.